x-pack/legacy/plugins/uptime/public/components/functional/monitor_list/monitor_list.tsx

```diff
@@ -105,6 +105,10 @@ export const MonitorListComponent = (props: Props) => {
         </EuiTitle>
         <EuiSpacer size="s" />
         <EuiBasicTable
+          aria-label={i18n.translate('xpack.uptime.monitorList.table.description', {
+            defaultMessage:
+              'Monitor Status table with 5 rows of data and columns for Status, Name, URL, IP, Downtime History and Integrations',
```

Comment:
`with 5 rows` seems misleading. There could be any number of rows, or the table could be empty.

Why don't we incorporate the number if items to the message?

Example:
```suggestion
          aria-label={i18n.translate('xpack.uptime.monitorList.table.description', {
            defaultMessage:
              'Monitor Status table with columns for Status, Name, URL, IP, Downtime History and Integrations. The table is currently displaying {length} items.',
            values: { length: items.length },
          })}
```

Reply:
oopss, that's a genuine mistake by me :)